lib/dry/cli.rb

```diff
@@ -154,11 +172,12 @@ def anonymous_registry(&block)
 
   # Create a new instance
   #
-  # @param registry [Dry::CLI::Registry, Block] a registry or a configuration block
+  # @param registry_or_command [Dry::CLI::Registry, Dry::CLI::Command] a registry or singular command
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line is too long. [101/100]](https://app.codacy.com/gh/dry-rb/dry-cli/pullRequest?prid=4800973)